xee/ext.py

```diff
@@ -139,7 +139,7 @@ def open(
       crs: Optional[str] = None,
       scale: Optional[float] = None,
       projection: Optional[ee.Projection] = None,
-      geometry: Optional[ee.Geometry] = None,
+      geometry: Optional[Union[ee.Geometry, types.BBox]] = None,
```

Comment:
Consider just using a union with None. Optional is like a Union, so this is somewhat redundant.

Reply:
Maybe you misread or I am misunderstanding. It sounds like you think I put `Optional[Union[ee.Geometry, None]]` in which case, yes, agreed--redundant, but I added types.BBox. I actually am not sure if the type `types.BBox` will display as tuple[float, float, float, float] in jupyter or something and I am somewhat inclined to change to `Optional[Union[ee.Geometry, Tuple[float, float, float, float]] = None` to make it as clear as possible.

Reply:
Implemented in https://github.com/google/Xee/pull/138/commits/4c0dd6b5bce2aacc3fbe44f7e6d8377ebf9a3971

Reply:
`Union[ee.Geometry, types.BBox, None]`  is equivalent to `Optional[Union[ee.Geometry, types.BBox]]` and in my opinion a bit cleaner.

Reply:
Sorry for being unclear. I'm referring to the definition of Optional: https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
I think I was confused, but your preference is clear now! Thanks. Updated in https://github.com/google/Xee/pull/138/commits/23ea0c9b402f775c18874e47a8a7283929b5b070